lib/errorHandlers/index.ts

```diff
@@ -41,6 +43,20 @@ export function logError(error: unknown, context?: ApiErrorContext): void {
     // Unknown errors
     logger.error(i18n(`${i18nKey}.unknownErrorOccurred`));
   }
+
+  if (isHubSpotHttpError(error) && error.code === 'ETIMEDOUT') {
+    const config = getConfig();
+    const defaultTimeout = config?.httpTimeout;
+
+    if (error.timeout && defaultTimeout === error.timeout) {
```

Comment:
Checking this to make sure the message doesn't get displayed in cases where we explicitly set a timeout

Reply:
Should we still have some sort of error override, even if we manually adjusted the timeout?

Reply:
Good question, I was thinking we'd just handle that on a case by case basis anywhere where we suspect it could be an issue (which is few or zero places as far as i know)

Reply:
Ah I see what you mean. My guess is that it would be fine to open this up to apply to all timeouts, but what you have is an improvement regardless. You can make the call to keep or remove the extra check

Reply:
Ohh I was just trying to avoid the case where we manually set a timeout somewhere and then a user tries to fix an error with `hs config set` but it doesn't work. I'll add a generic message for when that's not the case

Reply:
I didn't think about that. Then yeah +1 to keeping this in 👌

Reply:
Might be worth adding a comment so we don't forget that years from now lol